src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/ModuleScaffold.kt

```diff
@@ -607,4 +632,65 @@ object ModuleScaffold : Module("Scaffold", Category.WORLD) {
         return hasBlockInMainHand
     }
 
+    @Suppress("unused")
+    private val onNetworkTick = handler<PlayerNetworkMovementTickEvent> { event ->
+        if (event.state == EventState.PRE) {
+            for(i in 1..expandLenght) {
+                val block = findBestBlockPlacementTarget(
+                    player.blockPos.add(
+                        (-sin(player.yaw.toRadians()) * i).toInt(),
+                        -(player.blockPos.y - placementY),
+                        (cos(player.yaw.toRadians()) * i).toInt()
+                    ),
```

Comment:
Pretty smart solution 👍🏻

Reply:
credit goes to izuna for that

Reply:
i think it should be expanding in the dirrection of player movement
also your impl doesn't support fractional expand lengths

Reply:
how could you misspell length 😭

Reply:
man fuck off

Reply:
now that someone actually decides to make a much requested feature you shit on their spelling smh